app/src/main/java/com/infomaniak/swisstransfer/ui/components/SwissTransferTextField.kt

```diff
@@ -72,6 +76,12 @@ fun SwissTransferTextField(
         unfocusedSupportingTextColor = SwissTransferTheme.colors.tertiaryTextColor,
     )
 
+    @Composable
+    fun getSupportingText(): (@Composable () -> Unit)? {
+        val displayText = if (text.isEmpty()) supportingText else errorMessage() ?: supportingText
```

Comment:
Could you add parenthesis or brackets in this `if / else / ?:` combination please ?

I feel that this way of writing the condition is error prone.
I don't know exactly which `displayText` will be chosen.

Reply:
``` kotlin
val displayText = (if (text.isEmpty()) supportingText else errorMessage()) ?: supportingText
```
or
``` kotlin
val displayText = if (text.isEmpty()) {
    supportingText
} else {
    errorMessage() ?: supportingText
}
```